README.md

```diff
@@ -2,29 +2,30 @@
 
 # Neptune + Kedro integration
 
-Kedro plugin for experiment tracking and metadata management. It lets you browse, filter and sort runs in a nice UI, visualize node/pipeline metadata, and compare pipelines.
+Neptune's Kedro plugin for experiment tracking and metadata management. It lets you browse, filter and sort runs in a nice UI, visualize node/pipeline metadata, and compare pipelines.
 
 ## What will you get with this integration?
 
+* **realtime** pipeline execution status
 * **browse, filter, and sort** your model training runs
 * **compare nodes and pipelines** on metrics, visual node outputs, and more
 * **display all pipeline metadata** including learning curves for metrics, plots, and images, rich media like video and audio or interactive visualizations from Plotly, Altair, or Bokeh
 * and do whatever else you would expect from a modern ML metadata store
 
-![kedro](https://github.com/neptune-ai/kedro-neptune/assets/41324509/81a2c9b0-cbb7-4120-b111-434d441f08a0)
-*Kedro pipeline metadata in custom dashboard in the Neptune web app*
+![kedro](https://github.com/neptune-ai/kedro-neptune/assets/41324509/6551d7fd-0b0a-4589-8cee-d31fcdc7329a)
+*Kedro pipeline metadata in a custom dashboard in the Neptune web app*
 
 ## Resources
 
 * [Documentation](https://docs.neptune.ai/integrations/kedro)
 * [Code example on GitHub](https://github.com/neptune-ai/examples/tree/main/integrations-and-supported-tools/kedro/scripts/spaceflights)
-* [Example run logged in the Neptune app](https://app.neptune.ai/o/common/org/kedro-integration/runs/details?viewId=standard-view&detailsTab=dashboard&dashboardId=Spaceflights-42874940-da74-4cdc-94a4-315a7cdfbfa8&shortId=KED-2098)
+* [Example run logged in the Neptune app]([https://app.neptune.ai/o/common/org/kedro-integration/runs/details?viewId=standard-view&detailsTab=dashboard&dashboardId=Spaceflights-42874940-da74-4cdc-94a4-315a7cdfbfa8&shortId=KED-2098](https://app.neptune.ai/o/showcase/org/kedro/runs/details?viewId=9c348faf-42f3-4a6b-b5fd-2f7fc74b4c78&detailsTab=dashboard&dashboardId=Spaceflights-9b5f8098-9132-410f-a93a-0ab355a11fa5&shortId=KED-8))
 
 ## Support
 
 If you got stuck or simply want to talk to us, here are your options:
 
 * Check our [FAQ page](https://docs.neptune.ai/getting_help)
-* You can submit bug reports, feature requests, or contributions directly to the repository.
-* Chat! When in the Neptune application click on the blue message icon in the bottom-right corner and send a message. A real person will talk to you ASAP (typically very ASAP),
+* You can submit bug reports, feature requests, or contributions directly to the repository
+* Chat! When in the Neptune application, click on the blue message icon in the bottom-right corner and send a message. A real person will talk to you ASAP (typically very ASAP)
 * You can just shoot us an email at support@neptune.ai
```

Comment:
We have some complete sentences here, so let's instead end each list item with a period.